### PR TITLE
Feature/cc 0014 price bar

### PR DESCRIPTION
**Pull request by Fuchih**
**open**
OPEN

**close**
close

**resposive**
resposive

**hover**
Hover

**active**
active

### CODE REVIEW

**Review by tatoMa — CHANGES_REQUESTED** [review]
Also please double-check with BA and UI designer if this component is the only one showing on the screen with the gray shadow border. Looks like all layout components have different design styles now.
image

File: src/components/PriceBar/TireColorBoard.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+          inset -1px -1px 5px rgba(255, 255, 255, .9),
+          inset 1px 1px 5px rgba(0, 0, 0, .4);"
+        >
+          <Flex wrap="wrap" gap="4px" w="56px" h="35px" marginLeft="8px">
+            {mockTireColors.map((color) => (
+              <Box key={color} bg={color} w="16px" h="16px" />
+            ))}
+          </Flex>
+          <Text
+            w="105px"
```
Comment (tatoMa) [inline src/components/PriceBar/TireColorBoard.tsx:43]: May need to increase the width of this text component to make the sentence in 2 lines for matching the UI design.

File: src/components/PriceBar/index.tsx [hunk 2]
```diff
@@ -1,79 +1,62 @@
...
+            marginLeft: "98px",
+            marginBottom: "40px",
+          }}
+        >
+          {drawerButton(onClose)}
+          <DrawerBody boxShadow="0px -5px 10px -5px lightgrey" p="0">
+            <Flex>
+              <CourtTemplate />
+              <Spacer />
+              <TireColorBoard />
```
[the remark this answers is not in the document]
Comment (Fuchih) [reply src/components/PriceBar/index.tsx:54]: shold be tile

**Comment by EmilyGong111** [comment]
Screen Shot 2022-05-20 at 3 59 45 pm
The Text element has 3 lines, which is different with UI. May need a min-width to keep the sentence in 2 lines.

**Comment by Fuchih** [comment]
> Also please double-check with BA and UI designer if this component is the only one showing on the screen with the gray shadow border. Looks like all layout components have different design styles now. image

Hi tatoMa:
Yes, according to BA and UI,  it is the only component that has gray shadow border for now
@tatoMa

File: src/components/PriceBar/CourtTemplate.tsx [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+          >
+            {court.name}
+          </Text>
+        </Flex>
+      </Box>
+    );
+  });
+
+  return <Box display="flex">{listCourts}</Box>;
+};
```
Comment (tatoMa) [inline src/components/PriceBar/CourtTemplate.tsx:52]: It seems like you have 2 components nested with each other in one file. I would recommend to divide them into 2 separate files with passing props through.

**Review by tatoMa — CHANGES_REQUESTED** [review]
Please change the code to the pic I attached below. This is to fix the animation/transition of the drawer layer higher than footer problem.

**Review by tatoMa — APPROVED** [review]

**Comment by tatoMa** [comment]
LGTM

**Review by Jzeva — APPROVED** [review]

**Pull request merged**